inbm-vision/Changelog.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## NEXT - ?
 
+### Fixed
+- HSD 15010509095 - INBC fail to return correct Exit Code for difference scenario (-4/-6/-11)
+
 ### Fixed
```

Comment:
We still have duplicate Fixed tags for the next release.  Move the HSD entry to be with the others in the Fixed tag below.